packages/api/src/controllers/signing-key.ts

```diff
@@ -0,0 +1,238 @@
+import { authorizer, validatePost } from "../middleware";
+import { Router } from "express";
+import {
+  FieldsMap,
+  makeNextHREF,
+  parseFilters,
+  parseOrder,
+  toStringValues,
+} from "./helpers";
+import { db } from "../store";
+import sql from "sql-template-strings";
+import { v4 as uuid } from "uuid";
+import { generateKeyPairSync } from "crypto";
+import { ForbiddenError, NotFoundError } from "../store/errors";
+import {
+  SigningKey,
+  SigningKeyPatchPayload,
+  SigningKeyResponsePayload,
+} from "../schema/types";
+import { WithID } from "../store/types";
+
+const fieldsMap: FieldsMap = {
+  id: `signing_key.ID`,
+  name: { val: `signing_key.data->>'name'`, type: "full-text" },
+  deleted: { val: `signing_key.data->'deleted'`, type: "boolean" },
+  createdAt: { val: `signing_key.data->'createdAt'`, type: "int" },
+  userId: `signing_key.data->>'userId'`,
+};
+
+function generateSigningKeys() {
+  const keypair = generateKeyPairSync("rsa", {
+    modulusLength: 4096,
+    publicKeyEncoding: {
+      type: "spki",
+      format: "pem",
+    },
+    privateKeyEncoding: {
+      type: "pkcs8",
+      format: "pem",
+    },
+  });
+  return keypair;
+}
+
+const app = Router();
+
+app.get("/", authorizer({}), async (req, res) => {
```

Comment:
Maybe add a test that creates a signing key and lists all keys for this codecov comments to go away 😅 

A good test to do there is making sure users can't list keys from other users.